python/paddle/nn/functional/loss.py

```diff
@@ -3460,6 +3460,114 @@ def triplet_margin_loss(input,
         return loss
 
 
+def multi_margin_loss(input,
+                      label,
+                      p: int = 1,
+                      margin: float = 1.0,
+                      weight=None,
+                      reduction='mean',
+                      name=None):
+    r"""
+        Measures a multi-class classification hinge loss (margin-based loss) between input :math:`x` (a 2D mini-batch `Tensor`)
+        and output :math:`y` (which is a 1D tensor of target class indices,:math:`0 \leq y \leq \text{x.size}(1)-1`):
+
+        For each mini-batch sample, the loss in terms of the 1D input :math:`x` and scalar
+        output :math:`y` is:
+
+        .. math::
+            \text{loss}(x, y) = \frac{\sum_i \max(0, \text{margin} - x[y] + x[i])^p}{\text{x.size}(0)}
```

Comment:
`x.size(0)` is not a valid expression for Tensors in paddle.

Reply:
Done